io.github.Predidit.Kazumi.yaml

```diff
@@ -0,0 +1,104 @@
+id: io.github.Predidit.Kazumi
+# This app needs libwebkit2gtk-4.1 to work properly.
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: kazumi
+finish-args:
+  # x11 needs ipc permission.
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  # Flutter needs GPU permission.
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+
+modules:
+  - name: libayatana-appindicator
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DENABLE_BINDINGS_MONO=NO
+      - -DENABLE_BINDINGS_VALA=NO
+      - -DENABLE_GTKDOC=NO
+    cleanup:
+      - /lib/debug
+    sources:
+      - type: git
+        url: https://github.com/AyatanaIndicators/libayatana-appindicator.git
+        tag: 0.5.93
+        commit: 238c8b02718fa5b4af95ede72beeed762094f4cc
+        x-checker-data:
+          type: git
+    modules:
+      - shared-modules/intltool/intltool-0.51.json
+      - name: ayatana-ido
+        buildsystem: cmake-ninja
+        sources:
+          - type: git
+            url: https://github.com/AyatanaIndicators/ayatana-ido.git
+            tag: 0.10.4
+      - name: libayatana-indicator
+        buildsystem: cmake-ninja
+        config-opts:
+          - -DENABLE_LOADER=NO
+        cleanup:
+          - /lib/debug
+        sources:
+          - type: git
+            url: https://github.com/AyatanaIndicators/libayatana-indicator.git
+            tag: 0.9.4
+            commit: 611bb384b73fa6311777ba4c41381a06f5b99dad
+            x-checker-data:
+              type: git
+      - name: libdbusmenu-gtk3
+        buildsystem: autotools
+        build-options:
+          cflags: -Wno-error
+        config-opts:
+          - --with-gtk=3
+          - --disable-dumper
+          - --disable-static
+          - --disable-gtk-doc
+          - --enable-introspection=no
+          - --disable-vala
+        cleanup:
+          - /share/gtk-doc
+          - /share/libdbusmenu
+          - /lib/debug
+        sources:
+          - type: archive
+            url: https://launchpad.net/libdbusmenu/16.04/16.04.0/+download/libdbusmenu-16.04.0.tar.gz
+            sha256: b9cc4a2acd74509435892823607d966d424bd9ad5d0b00938f27240a1bfa878a
+  - name: kazumi
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/Predidit/Kazumi/releases/download/1.3.0/Kazumi_linux_1.3.0_amd64.deb
+        sha256: 599b6cc234ebdf9d2825656aae1a8591d70b7764d60010aee63c682bcc51c14e
+        only-arches: [x86_64]
```

Comment:
no aarch64 version? I thought "Flutter" was supposed to be cross-platform...

Reply:
I am asking the upstream developer to do it. It only has x86_64 for now.